python/PiFinder/imu_pi.py

```diff
@@ -20,8 +20,8 @@ class Imu:
     def __init__(self):
         i2c = board.I2C()
         self.sensor = adafruit_bno055.BNO055_I2C(i2c)
-        # self.sensor.mode = adafruit_bno055.IMUPLUS_MODE
-        self.sensor.mode = adafruit_bno055.NDOF_MODE
+        self.sensor.mode = adafruit_bno055.IMUPLUS_MODE
```

Comment:
I'm still evaluating both of these options... did you see  any difference between them in your usage?